pkg/controller/actuator.go

```diff
@@ -202,6 +202,9 @@ func (a *actuator) createIssuerValues(cluster *controller.Cluster, issuers ...se
 		if issuer.RequestsPerDayQuota != nil {
 			issuerValues["requestsPerDayQuota"] = *issuer.RequestsPerDayQuota
 		}
+		if len(issuer.PrecheckNameservers) > 0 {
+			issuerValues["precheckNameservers"] = issuer.PrecheckNameservers
```

Comment:
Does it make sense to validate the input of `issuer.PrecheckNameservers` or is this done in `gardener/cert-management`, so that the issuer won't get ready?
My concern is that w/o validation we potentially add more ops effort: Shoot owners configure issuers, reconciliation finishes successfully, but shoot "users" notice an issue later when a certificate cannot be issued.